.gitignore

```diff
@@ -1,6 +1,3 @@
-binaries/*
-!binaries/2.9.2
```

Comment:
why?

Reply:
because there is no longer a need for this as the update binaries bash script removes the folder. This was originally made to not bundle multiple versions together.

Reply:
in this case, I would modify the script to edit the .gitignore file as well, since the script is not the only way to push new binaries, and .gitignore should protect multiple binaries being pushed by mistake

Reply:
💩

Reply:
This is the worst manual step ever needed

Reply:
@amircodota

Reply:
sed -i 's/old_version/new_version/' filename

Reply:
currentversion=$(grep -Eo '!binaries/.*' .gitignore | cut -c10-)

Reply:
I have no idea how to make it work. I reverted.